src/utils/useFetchGet.ts

```diff
@@ -0,0 +1,53 @@
+import {useState, useEffect} from 'react'
+
+const useFetchGet = (url:string) => {
```

Comment:
as discussed, please do not introduce this hook, and instead create a dedicated hook for each thing that needs fetching inside of `DashboardController`. The primary benefit of doing so is that consumers of the hook will have type safety (the return value will be `ComputeService[]` or whatever, rather than `any[]`

Reply:
Updated